aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/BaseHandlerStd.java

```diff
@@ -457,7 +457,7 @@ protected void removeOldTags(
                             UPDATE_ASSOCIATED_ROLES_ERROR_RULE_SET
                     ))
                     .success();
-            if (!progressEvent.isSuccess()) {
+            if (progressEvent.isFailed()) {
                 return progressEvent;
```

Comment:
same as in the other pr. I dont think this is right, we need to return also if it's in_progress. we basically can ignore only success bc the rest we need to propagate back to cfn